### PR TITLE
Add examples for smaller GPUs

### LINKED ISSUE #14: Run on less than 8 GB vram
Please describe how to run it on GPU with less than 8 GB VRAM

### PR DESCRIPTION
**Pull request by fboulnois**
Add additional details and examples when using smaller GPUs. Resolves #14 , resolves #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +91,17 @@ Options can be combined:
 ./build.sh run --scale 7.0 --seed 42 'abstract art'
 ```
 
+On systems with <8GB of GPU RAM, you can try mixing and matching options:
+
+* Make images smaller than 512x512 using `--W` and `--H`
```
Comment (MicahZoltu) [inline README.md]: Probably worth noting either here or in the options section, that the width and height need to be divisible by 64 (I think, this worked for me while simple multiples of 8 didn't).

File: README.md [hunk 2]
```diff
@@ -90,6 +91,17 @@ Options can be combined:
...
+* Use `--half` to decrease memory use but slightly decrease image quality
+* Use `--attention-slicing` to decrease memory use but decrease image speed
```
Comment (MicahZoltu) [inline README.md]:
```suggestion
* Use `--attention-slicing` to decrease memory use but also decrease image generation speed
```

File: README.md (hunk 1 above)
Comment (MicahZoltu) [inline README.md]:
```suggestion
* Make images smaller than 512x512 using `--W` and `--H`, which proportionately reduces memory usage and increases generation speed
```

**Review by MicahZoltu — APPROVED** [review]
Closes #16 and #14.

I left some minor feedback, I think this is worth merging with or without those suggestions.

Thanks!

File: README.md (hunk 1 above)
Comment (fboulnois) [reply README.md]: I added this to the option descriptions.

Comment (fboulnois) [reply README.md]: I'm not sure if the change is precisely **proportional**, so I omitted that part of the description.

File: README.md (hunk 2 above)
Comment (fboulnois) [reply README.md]: I used the word `creation` since `create` is used in the rest of the README.

**Pull request merged**